apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -12,7 +13,7 @@ if (!('BridgeServiceMixin' in self)) {
 (function(exports) {
 
 const SERVICE_NAME = 'messaging-service';
-const MOBILE_MESSAGE_CLIENT_NAME = 'mozMobileMessageShim';
+const MOBILE_MESSAGE_CLIENT_NAME = 'moz-mobile-message-shim';
```

Comment:
__note:__ it doesn't really matter, but I don't feel confident about dash-based shim name, I'd keep it camelCase as we do for other services/shims. I suspect it was done to reuse it for the ```BroadcastChannel``` name, but we'll likely have more complex channel name like ```appInstanceId +'moz-mobile-message-shim'``` to avoid collisions between several app instances.... And with current approach it will look a bit monstrous :) - ```appInstanceId + MOBILE_MESSAGE_CLIENT_NAME + '-channel'```. But it's up to you in the end.